cluster/gce/addons/loadbalancing/cloud-provider-binding.yaml

```diff
@@ -18,7 +50,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
```

Comment:
Actually, I'm not sure there's any reason we need to keep the old ClusterRoleBinding. The new one grants a superset of the permissions.

I think we just need to keep the old cluster role incase someone was binding it to a different service account.

Reply:
lol Good point. No, there's no reason to keep the bindings. Only the old roles need to be remained.